adam/curriculum/phase1_curriculum.py

```diff
@@ -1559,35 +1565,107 @@ def make_behind_in_front_templates() -> Iterable[Phase1SituationTemplate]:
     )
 
 
+# Simple Pursuit curriculum
+def _make_simple_pursuit_curriculum(
+    n_exemplar: int = 10, n_noise: int = 0, n_objects_in_scene: int = 3
+) -> Phase1InstanceGroup:
+    if n_noise > n_exemplar:
+        raise RuntimeError("Cannot have more noise than regular exemplars")
+
+    target_objects = [BALL, CHAIR, PERSON, TABLE, DOG, BIRD, BOX]
+    target_object_variables = [
+        object_variable(target.handle + "-target", target) for target in target_objects
+    ]
+    other_objects = [
+        standard_object("obj-" + str(idx)) for idx in range(n_objects_in_scene)
+    ]
+
+    simple_pursuit_templates = [
+        Phase1SituationTemplate(
+            "simple_pursuit",
+            salient_object_variables=[target_object_variable],
+            background_object_variables=other_objects[:-1],
+        )
+        for target_object_variable in target_object_variables
+    ]
+
+    noise_template = Phase1SituationTemplate(
+        "simple_pursuit",
+        salient_object_variables=[other_objects[0]],
+        background_object_variables=other_objects[1:],
+    )
+
+    all_instances = []
+    # Generate phase_1 instance groups for each template (i.e each target word)
+    for template in simple_pursuit_templates:
+        new_instances = list(
+            phase1_instances(
+                "simple_pursuit_curriculum",
+                sampled(
+                    template,
+                    max_to_sample=n_exemplar - n_noise,
+                    chooser=PHASE1_CHOOSER,
+                    ontology=GAILA_PHASE_1_ONTOLOGY,
+                ),
+            ).instances()
+        )
+        all_instances.extend(new_instances)
+        # Create instances for noise
+        if n_noise > 0:
+            noise_instances = phase1_instances(
+                "simple_pursuit_curriculum",
+                sampled(
+                    noise_template,
+                    max_to_sample=n_noise,
+                    chooser=PHASE1_CHOOSER,
+                    ontology=GAILA_PHASE_1_ONTOLOGY,
+                ),
+            ).instances()
+            for noise_instance in noise_instances:
+                # Get the correct language dependency tree
+                dependency_tree = PHASE1_CHOOSER.choice(new_instances)[1]
+                all_instances.append(
+                    (noise_instance[0], dependency_tree, noise_instance[2])
+                )
+
+    description = f"simple_pursuit_curriculum_examples-{n_exemplar}_objects-{n_objects_in_scene}_noise-{n_noise}"
+    final_instance_group: Phase1InstanceGroup = ExplicitWithSituationInstanceGroup(
+        description, all_instances
+    )
+    return final_instance_group
+
+
 GAILA_PHASE_1_CURRICULUM = [
```

Comment:
Am I misreading this, or does this *replace* the normal phase 1 curriculum with the pursuit curriculum?

Reply:
I'd commented those out for faster testing - thank you for catching!